client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/AdministratorController.java

```diff
@@ -123,7 +126,33 @@ private void initPermissionCombobox(){
        ObservableList<String> permissions = 
                FXCollections.observableArrayList(Permission.ADMIN.toString(),
                Permission.EDITOR.toString(), Permission.VIEWER.toString());
-       permissionCombobox = new ComboBox<String>(permissions);
+       permissionCombobox.setItems(permissions);
+    }
+    
+    public void populateTable(){
+        Platform.runLater(new Runnable() {
+            
+            @Override
+            public void run() {
+                staffList = FXCollections.observableArrayList();
+                List<Staff> users = administratorService.getInspectors();
+                if(users!=null){
+                    for(Staff s : users){
+                        staffList.add(s);
+                    }
+                    
+                    staffTable.setItems(staffList);
+                }
```

Comment:
How is this easier that simply using:

    staffList = FXCollections.observableArrayList(administrationService.getInspectors();
    staffTable.setItems(staffList);

?